src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -131,16 +155,15 @@ const PatientProfilePage = ({
         hospitalId: Hospital;
         ambulanceId: Ambulance;
       } = data.patient;
-      setFormFields({
-        ...formFields,
-        barcodeValue,
+      setFormFields(() => ({
+        barcodeValue: barcodeValue,
         triage: triageLevel,
-        gender,
-        age,
-        notes,
-        status,
-        runNumber,
-      });
+        gender: gender,
+        age: age,
+        notes: notes,
+        status: status,
+        runNumber: runNumber,
+      }));
```

Comment:
Out of curiosity, why did you make these updates? Should make no difference because we were just using the [shorthand](https://alligator.io/js/object-property-shorthand-es6/) before.

Reply:
Just checked and it actually makes our linter unhappy :(

Reply:
Ok let me walk through my logic on why I made this change:
1. We want to get rid of the warnings with `useEffect()` complaining about unneeded dependencies. We can either add the dependency or we can refactor
2. Adding the dependency leads to infinite loops. Specifically, this warning is complaining that `formFields` should be a dependency but the hook itself changes `formFields`...
3. To refactor, we need to remove all instances of `formFields`. I thought the best way of doing this is the functional `setFormFields` that I wrote
4. It seems to have passed our linter. I did notice that the linter complains but linting it causes no issues.

Reply:
i think michelle is referring specifically to the prop assignment shorthand (i.e. using `gender: gender` when `gender` is enough)

can either of you explain what you mean by the linter complaining? it seems fine on my end (e.g. no errors or warnings show up)

Reply:
I get red lines on VS Code regarding property shorthands from aslant

Reply:
@aaronabraham311 do you have issues with the shorthand syntax? if vscode is complaining, i'd prefer to stick with the shorthand if possible

Reply:
i realize that the linter actually fixes this syntax and makes it into the shorthand, so let's revert this so it just goes with the original shorthand